composer.json

```diff
@@ -17,6 +17,7 @@
     },
     "require": {
         "php": ">=5.4.0",
+        "ext-xdebug": "*",
```

Comment:
Right! I got `2.2.7` installed and it seems like `2.2.0` was released back in 2012. `~2.2` should do the trick.